gulpfile.js

```diff
@@ -74,22 +74,37 @@ const autoGeneratedHeader = `
 // standard themes css compilation
 function styles_style_portal() {
     return gulp.src(config.src.styles.style_portal)
-    .pipe(injector.replace('// bs4import', '@import "../../public/assets/bootstrap/scss/bootstrap";'))
-    .pipe(sourcemaps.init())
-    .pipe(sass().on('error', sass.logError))
-    .pipe(postcss([ prefix() ]))
-    .pipe(gap.prependText(autoGeneratedHeader))
-    .pipe(gulpif(!config.dev, csso()))
-    .pipe(gulpif(!config.dev, sourcemaps.write()))
-    .pipe(gulp.dest(config.dest.themes));
+        .pipe(injector.replace('// bs4import', '@import "../../public/assets/bootstrap/scss/bootstrap";'))
+        .pipe(sourcemaps.init())
+        .pipe(sass().on('error', sass.logError))
+        .pipe(postcss([prefix()]))
+        .pipe(gap.prependText(autoGeneratedHeader))
+        .pipe(gulpif(!config.dev, csso()))
+        .pipe(gulpif(!config.dev, sourcemaps.write()))
+        .pipe(gulp.dest(config.dest.themes));
 }
 // standard themes css compilation
 function styles_style_uni() {
+    // First, create the compact version
+    gulp.src(config.src.styles.style_uni)
```

Comment:
this is increasing build time about 2.5x . Just making sure these new commands are async (I am assuming they are, just want to verify).

Reply:
This is async as each theme file creates a compact version followed by a regular version. This created another issue for me which is the fact that creating a compact version of each one is basically the same as compiling all the themes one more time therefore increasing compile time. I'll have to find a more elegant solution for this but for now the build time will be a bit long until I find another solution.

Reply:
Or at least I think it's async...

Reply:
i was able to get minimal improvement (from 4:30 to 4:10) if break these into their own functions/calls. I'll do some more testing on that and plan to bring it in after this PR (will bring this in after silent travis issues are fixed :) ).